.gitlab-ci.yml

```diff
@@ -53,3 +63,8 @@ test-3.6:
   extends: .linux-test-base
   variables:
     PYTHON_VERSION: "3.6"
+
+test-3.6-win:
+  extends: .windows-test-base
+  tags:
+    - windows
```

Comment:
Please add a new line to the end of the file.

Reply:
resolved